### PR TITLE
Use dfe-analytics

### PR DESCRIPTION
**Pull request by duncanjbrown**
### Context

Replace this app's custom BigQuery integration with the `dfe-analytics` gem

### CODE REVIEW

**Comment by duncanjbrown** [comment]
Deploying this to a review app so we can test it works 😅

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN apk add --update --no-cache tzdata && \
     echo "Europe/London" > /etc/timezone
 
 RUN apk add --update --no-cache --virtual runtime-dependances \
-    yarn shared-mime-info
+    yarn shared-mime-info git
```
Comment (misaka) [inline Dockerfile:10]: What is this needed in the Docker container for? I can't see where git is used there ...

Comment (duncanjbrown) [reply Dockerfile:10]: Because we're pulling from GitHub until we get RubyGems

Comment (misaka) [reply Dockerfile:10]: Ah, for when we run `bundle` in the Docker container. Of course. 👍

**Review by misaka — APPROVED** [review]

**Pull request merged**